stumpy/core.py

```diff
@@ -383,7 +383,7 @@ def are_arrays_equal(a, b):  # pragma: no cover
     if a.shape != b.shape:
         return False
 
-    return ((a == b) | (np.isnan(a) & np.isnan(b))).all()
+    return bool(((a == b) | (np.isnan(a) & np.isnan(b))).all())
```

Comment:
Cool. I think `bool(...)` should return `False` that points to the same memory where the Boolean value `False` is located.